PF/PF-Assignment1/Cart.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
Try to use basic data structures like the array in starting assignments. We will move towards inbuild DS after data structure session.